.travis.yml

```diff
@@ -5,15 +5,26 @@ env:
 install:
   - pip install GitPython
   - export TRAVIS_COMMIT_MSG="$TRAVIS_REPO_SLUG - $(git log --format=%B --no-merges -n 1)"
-  - git clone https://github.com/xbmc-catchuptv-au/repo-devel.git .deploy
 script:
-  - cd .deploy
-  - ./manage_repo.py ../
+  - 'if [ -z "$TRAVIS_TAG" ]; then
+         git clone https://github.com/glennguy/repo-devel.git .deploy || travis_terminate 1;
+         cd .deploy || travis_terminate 1;
+         ./manage_repo.py ../ || travis_terminate 1;
+     else
```

Comment:
When we push a tag, we should also build the same version in the devel repo too.

Reply:
Sure thing, will do!